code/renderers/react/src/enrichCsf.ts

```diff
@@ -18,17 +18,21 @@ export const enrichCsf: PresetPropertyFn<'experimental_enrichCsf'> = async (inpu
         return;
       }
       const { format } = await getPrettier();
+      let node;
+      try {
+        node = getCodeSnippet(storyExport, csfSource._metaNode, csfSource._meta?.component);
+      } catch (e) {
+        // don't bother the user if we can't generate a snippet
```

Comment:
maybe use `logger.debug` here from `storybook/internal/node-logger` so it's at least easy to get this information if we need to?